src/utils/migrations/initialize-ark-networks.ts

```diff
@@ -1,21 +1,12 @@
 import { ARK } from "@ardenthq/sdk-ark";
-import { isE2E, isPreview } from "@/utils/test-helpers";
 
 export const initializeArkNetworks = ({ data }) => {
-	if (typeof data.networks === "object" && !!data.networks.ark) {
-		// Networks already assigned to profile, skipping migration
-		return;
-	}
-
 	if (!data.networks) {
 		data.networks = {};
 	}
 
 	data.networks.ark = {
+		devnet: ARK.manifest.networks["ark.devnet"],
```

Comment:
would this cause devnet to be enabled by default in production or does it not affect that?